getting-started/deployment_guide.md

```diff
@@ -90,6 +91,7 @@ karmor log
 3. Google Kubernetes Engine (GKE) with Container Optimized OS (COS)
 4. GKE with Ubuntu image
 5. [Amazon Elastic Kubernetes Service (EKS)](../deployments/EKS)
+5. Azure Kubernetes Service (AKS)
```

Comment:
Shall we add a hyperlink for the docs?

Reply:
Can you add a document showing how spin up a AKS cluster using the given OS image etc? We need to let users know what was the exact config with which the cluster was spawned.

Reply:
yes, it would be better to have a document for AKS (similar to EKS).

Reply:
@nyrahul @nam-jaehyun Sure, I'll add a link showing how users could create an AKS cluster.
Also, I think OS won't be a problem here as AKS doesn't really give users the choice to use any other linux distros other than the default Ubuntu 18.04.